src/sign/mod.rs

```diff
@@ -1,10 +1,57 @@
 //! DNSSEC signing.
 //!
 //! **This module is experimental and likely to change significantly.**
+//!
+//! Signatures are at the heart of DNSSEC -- they confirm the authenticity of a
+//! DNS record served by a secure-aware name server.  But name servers are not
+//! usually creating those signatures themselves.  Within a DNS zone, it is the
```

Comment:
I'm not sure what you mean by a name server not signing a zone itself. Some name servers do indeed do this themselves.

Reply:
I didn't phrase it well -- I meant that signing seems to usually be done offline.  I'll update that if online signing is a common convention.